### ISSUE TITLE
[BUG] Getting started-movielens 03-Training-with-HugeCTR.ipynb nb does not print out the acc and loss metrics

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/NVTabular#1521]
**Describe the bug**
When I run the  [03-Training-with-HugeCTR.ipynb](https://github.com/NVIDIA-Merlin/NVTabular/blob/main/examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb) notebook, the training step does not print out the accuracy and loss metrics.

We can avoid this 

1. Move the following commands to another cell:

```
model.compile()
model.summary()
model.fit(max_iter=2000, display=100, eval_interval=200, snapshot=1900)
model.graph_to_json(graph_config_file=MODEL_DIR + "1/movielens.json")
```

OR

2. add the training commands to a  `.py` file and run the following in another cell:
```
!python model.py
```

Second option might be problematic for the hugectr unit test because of `!`. 

**Steps/Code to reproduce bug**
please run [03-Training-with-HugeCTR.ipynb](https://github.com/NVIDIA-Merlin/NVTabular/blob/main/examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb)

**Expected behavior**

**Environment details (please complete the following information):**
 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
 - Method of NVTabular install: [conda, Docker, or from source]
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used

`merlin-training:22.04` container is used

**Comment by rnyak** [comment]
@benfred and @karlhigley can you please advice? I will create a PR accordingly. thanks.

**Comment by karlhigley** [comment]
The first option sounds good to me
